build.gradle

```diff
@@ -267,7 +267,7 @@ Boolean bwcBundleTest = (project.findProperty('customDistributionDownloadType')
 When testing between major versions note that ONLY the latest minor release of the previous version is api compatible. 
 */
 
-String bwcVersion = "2.15.0.0"
+String bwcVersion = "2.16.0.0"
```

Comment:
Should this be bumped to 2.17 now that core has bumped the version?

Reply:
You're right this pr is very stale. Thank you for the comment!